posix/posix.c

```diff
@@ -1147,10 +1147,10 @@ int posix_ftruncate(int fildes, off_t length)
 
 	err = posix_getOpenFile(fildes, &f);
 	if (err == 0) {
```

Comment:
SHould'nt it be >= 0?

Reply:
`posix_getOpenFile` returns 0 or minus values, but why not, I guess